samples/military_feature_schemas/Fields_Air.csv

```diff
@@ -0,0 +1,30 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain,field_default,field_notes
+identity,Integer,4,Standard Identity,NULLABLE,Identity,,
+context,SmallInteger,2,Context,NULLABLE,Context,,
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,Air,1,
+entity,Integer,4,Air Vehicle Type Icon,NULLABLE,Air_Entities,,
+modifier1,Integer,4,Modifier 1,NULLABLE,Air_Modifier_Ones,,
+modifier2,Integer,4,Modifier 2,NULLABLE,Air_Modifier_Twos,,
+indicator,Integer,4,"HQ, Task Force, Feint/Dummy",NULLABLE,HQTFFD,,
+operationalcondition,Integer,4,Operational Condition,NULLABLE,Operational_Condition_Amplifier,,
+uniquedesignation,String,30,Unique Designation,NULLABLE,,,
+additionalinformation,String,50,Additional Information,NULLABLE,,,
+staffcomment,String,50,Staff Comments,NULLABLE,,,
+credibility,SmallInteger,2,Credibility,NULLABLE,Credibility,,
+reliability,String,2,Reliability,NULLABLE,Reliability,,
+idmode,String,15,IFF/SIF/AIS,NULLABLE,,,
+type,String,24,Type of Equipment,NULLABLE,,,
+direction,SmallInteger,2,Direction,NULLABLE,Direction,0,
+speed,SmallInteger,2,Speed,NULLABLE,,,
+speedunit,String,3,Speed Unit,NULLABLE,SpeedUnits,,
```

Comment:
2525D refers to MIL-STD-6040 (USMTF) when it describes how speed is to be drawn next to a symbol.  

![image](https://cloud.githubusercontent.com/assets/4564846/9769540/6e5d0b82-56f0-11e5-946e-dd0a12d76f11.png)

6040 is a "Distribution C" DOD document, therefore ITAR and LIMDIS controlled and not to be released to the public.  I was able to acquire just the relevant part of this document which I have now placed [here](https://esri.box.com/s/98c91ggsz9f9cv2u53uy6p4ixqvlkf8y): 

Without a speed unit field, we would never be able to communicate to our users that a particular speed value is MPH, KPH, MPS, or KTS.